jboss-kie-kieserver/added/launch/jboss-kie-kieserver-hooks.sh

```diff
@@ -56,8 +56,7 @@ if [ -n ${WORKBENCH_SERVICE_NAME} -a -n "${KIE_SERVER_ID}" ]; then
         fi
 
     elif [ ${kieReplicas} -gt 0 ]; then
-        log_info "KIE Server Replicas is ${kieReplicas}, updating ${KIE_SERVER_ID} configMap to USED."
-        update_config_map "{\"metadata\":{\"labels\":{\"services.server.kie.org/kie-server-state\":\"USED\"}}}" "${kieDCName}"
+        log_info "KIE Server Replicas is ${kieReplicas}, connecting ${KIE_SERVER_ID} to BC/WB if exists."
```

Comment:
Wouldn't it be necessary anymore?

Reply:
Keeping the info log which contains KieServer Id and number of replicas is mainly for better debug-ability.
The analysis for not updating CM label has been elaborated [here](https://issues.redhat.com/browse/RHPAM-2634?focusedCommentId=13947321&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13947321). It is unlikely to bring any negative side effects.